dashboards-observability/public/components/custom_panels/home.tsx

```diff
@@ -176,7 +176,7 @@ export const Home = ({ http, chrome, parentBreadcrumb, pplService, renderProps }
       });
```

Comment:
Should we add a filter in panels/home.tsx `in fetchCustomPanels function` , which will filter out the panels which are part of applicationID associated with them?

Reply:
Yes! I will be adding that in the next PR, didn't want to make it too long.